verilog/tools/formatter/verilog_format.cc

```diff
@@ -85,6 +85,8 @@ std::string AbslUnparseFlag(LineRanges /* unused */) {
 // TODO(fangism): Provide -i alias, as it is canonical to many formatters
 ABSL_FLAG(bool, inplace, false,
           "If true, overwrite the input file on successful conditions.");
+ABSL_FLAG(bool, check, false,
```

Comment:
I am slightly hesitant about using `--check` as flag name as we have the `CHECK()`-macro that crashes on an assert issue, so whenever I see 'check' I am associating it with a noisy failure :)

So I am mostly ok with the flag choice (say, 80%), but I'd like to summon @fangism if he has thoughts in this regard.

Other names could be `--verify`, `--just_verify`, `--verify_formatting`.

Reply:
Let me change to --verify